charts/aws-cloud-controller-manager/values.yaml

```diff
@@ -0,0 +1,116 @@
+namespace: "kube-system"
+
+extraArgs: {}
+
+
+image:
+    repository: gcr.io/k8s-staging-provider-aws/cloud-controller-manager
+    tag:  v1.19.0-alpha.1
+
+# nameOverride -- String to partially override `aws-cloud-controller-manager.fullname`
+nameOverride: "aws-cloud-controller-manager"
+
+# nodeSelector -- Node labels for pod assignment. Ref: https://kubernetes.io/docs/user-guide/node-selection/.
+
+nodeSelector:
+  node-role.kubernetes.io/master: ""
+
+clusterRoleRules:
```

Comment:
Can we have something like:

```
rbac:
    create: true

serviceAccount:
    create: true
```

In case users want to set it up a little differently or configure the RBAC to come from a different source?

Reply:
@nckturner Sure, You want to change this in the current version, or can be added later? Please let me know

Reply:
+1. This was requested multiple times for the csi drivers. We need to support existing service accounts.

Reply:
If you prefer to add later that's fine

Reply:
Sure @nckturner I will add a follow-up issue on this, Thanks :)